### ISSUE TITLE
 When I finished training with my own data set, I found that the model did not play any role, but the training process was not wrong, why?

### ISSUE THREAD

**Issue by guangmingdexin** [zzh8829/yolov3-tf2#229]

**Comment by guangmingdexin** [comment]
It cannot detect anything

**Comment by tsk15535904190** [comment]
I have such questions ,I use 300 samples of fire for train and 4 batch size and 20 epochs ,the last model is 400+Mb but i can't detect anything

**Comment by tsk15535904190** [comment]
you have to set lower the thershold ,I had successed

**Comment by gaofssvm** [comment]
I have the same questions ,I use 1200 samples of fire for train and 8 batch size and 60 epochs with pre_trianing  ,the last model is 1000+Mb but i can't detect anything,i also set lower the  thershold

**Comment by tsk15535904190** [comment]
> I have the same questions ,I use 1200 samples of fire for train and 8 batch size and 60 epochs with pre_trianing  ,the last model is 1000+Mb but i can't detect anything,i also set lower the  thershold

what's the final-loss? it's prefer to lower than 20

**Comment by gaofssvm** [comment]
yes，final val_loss is lower than 20

**Comment by guangmingdexin** [comment]
set lower ignore_threshold in the model.py loss function

**Comment by gaofssvm** [comment]
@guangmingdexin,thanks for you  ,i set  ignore_threshold lower to 0.1,and detect nothing

**Comment by w19787** [comment]
@guangmingdexin @tsk15535904190 @gaofssvm @burnpiro i submit a pull request to modify loss. I expect it to solve the above issue. if you can help to test whether it works for you will be deeply appreciated.
